src/java/us/kbase/templates/module_makefile.vm.properties

```diff
@@ -49,6 +41,8 @@ compile:
 #if($language == "java")
 		--javasrv \
 		--javapackage ${java_package_parent};
+		--javasrc src \
+		--java \
```

Comment:
Why generate java clients for java servers when we don't generate py clients for py servers etc?

Reply:
I left it in because I was not able to grok the purpose of that flag, but also I am fairly confident we can take it out as long as `--javasrv` is left

So the flag sets javaClientSide in ModuleBuilder and then is passed to here: https://github.com/kbase/kb_sdk/blob/master/src/java/us/kbase/mobu/compiler/RunCompileCommand.java#L151. javaClientSide will always be true with the --javasrv flag. It's unclear to me how any of that is connected to JavaTypeGenerator.processSpec.